include/deal.II/matrix_free/tensor_product_kernels.h

```diff
@@ -1253,59 +1242,34 @@ namespace internal
      * @tparam one_line If true, the kernel is only applied along a single 1d
      *                  stripe within a dim-dimensional tensor, not the full
      *                  n_rows^dim points as in the @p false case.
+     * @tparam quantity Specify whether values, gradients or Hessians should
+     *                  be interpolated, allowing specialized algorithms
+     *                  for some class template parameters of `variant` to
+     *                  find the right path.
+     * @tparam extra_stride This parameter enables to place the result of the
```

Comment:
`stride` vs. `extra_stride`: seem to be the same. If yes, could we use the same names?